score-web/src/app/account-management/domain/tenants.ts

```diff
@@ -0,0 +1,65 @@
+import {PageRequest} from '../../basis/basis';
+import {ParamMap} from '@angular/router';
+import {HttpParams} from '@angular/common/http';
+import {base64Decode, base64Encode} from '../../common/utility';
+
+export class TenantListRequest {
+    name: string;
+    page: PageRequest = new PageRequest();
+
+    constructor(paramMap?: ParamMap, defaultPageRequest?: PageRequest) {
+        const q = (paramMap) ? paramMap.get('q') : undefined;
+        const params = (q) ? new HttpParams({fromString: base64Decode(q)}) : new HttpParams(); 
+        this.page.sortActive = params.get('sortActive');
+        if (!this.page.sortActive) {
+          this.page.sortActive = (defaultPageRequest) ? defaultPageRequest.sortActive : '';
+        }
```

Comment:
Can you add a blank line after the closing bracket in such cases? It would make it a bit more readable.